vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -6,12 +6,16 @@
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "amd64OnlyVersions": [],
         "multiArchVersions": [
-          "1.24.9-hotfix.20230208.2",
           "1.24.10.3",
+          "1.24.9-hotfix.20230208.2",
           "1.25.5-hotfix.20230208",
+          "1.25.5-hotfix.20230407.1",
           "1.25.6",
+          "1.25.6-hotfix.20230407.1",
           "1.26.0",
-          "1.26.3.2"
+          "1.26.0-hotfix.20230407.1",
+          "1.26.3.2",
```

Comment:
currently in RP we are using 1.26.3.2 without baking it into VHD, but 1.26.3.4 is already available.